Automattic/wp-calypso
# Add feature flag for down time monitoring enhancements

Add a feature flag for the dashboard down time monitoring updates and enhancements project

#### Proposed Changes

* Adds the jetpack/partner-portal-downtime-monitoring-updates feature flag to be used in development of the dashboard enhancements and updates for the down time monitoring feature. 

#### Testing Instructions



* Checkout this branch
* Run `yarn-start-jetpack-cloud`
* Visit the Dashboard at http://jetpack.cloud.localhost:3000/dashboard
* Ensure that the Dashboard title is no longer "Dashboard" and is now "Dashboard - Down Time Monitoring Updates Enabled"
* Open the Jetpack Cloud Live (horizon) link below and visit the Dashboard home page. 
* Ensure that the Dashboard title is still "Dashboard" 
- Open the `config/jetpack-cloud-development.json` file and verify that `jetpack/partner-portal-downtime-monitoring-updates` flag is set to `true`
- Open the following files and verify that the value for `jetpack/partner-portal-downtime-monitoring-updates` flag is set to `false`:
  - config/jetpack-cloud-horizon.json
  - config/jetpack-cloud-production.json
  - config/jetpack-cloud-stage.json
<img width="958" alt="Screenshot 2022-12-08 at 10 55 19 AM" src="https://user-images.githubusercontent.com/1273880/206527940-8253e493-6015-4371-95f1-fecffbfa2b21.png">


#### Pre-merge Checklist





Related to #
